experimental/bot/logger/default_logger.go

```diff
@@ -25,6 +25,15 @@ New creates a new logger.
 - api: LogAPI implementation
 
 - options: Optional parameters for the Logger. Available options are MessageToAdmins and LogToTelemetry.
+
+Example:
+  client := pluginapi.NewClient(p.api)
+  dmer := poster.New(...)
+  tracker := telemetry.NewTracker(...)
+  logger := New(client,
+	MessageToAdmins(LogLevelWarn, false, dmer, "000000000000000", "0000000010000"),
+	LogToTelemetry(LogLevelDebug, tracker),
+  )
```

Comment:
Godoc supports documenting examples directly: https://golang.org/pkg/testing/#hdr-Examples. It's the idiomatic way to write examples.

Reply:
Noted. Nevertheless, it seems like there is no need for examples now. What do you think?

Reply:
This is defiantly a non-blocking suggestion. Still, I consider every example helpful and valuable.